geoips/compare_outputs.py

```diff
@@ -244,7 +244,9 @@ def images_match(output_product, compare_product, fuzz="5%"):
     ]
 
     LOG.info("**Running %s", " ".join(call_list))
-    fullimg_retval = subprocess.call(call_list)
+    fullimg_retval = subprocess.call(
```

Comment:
Added subprocess.PIPE so the output doesn't go to the terminal.  If we decide we want these back in the info level, we'll have to reimplement another way.